src/main/scala/com/gd/twitteranalytics/TwitterReportGenerator.scala

```diff
@@ -0,0 +1,60 @@
+package com.gd.twitteranalytics
+
+import com.gd.twitteranalytics.TwitterReportingJob.log
+import org.apache.spark.sql.expressions.Window
+import org.apache.spark.sql.functions.count
+import org.apache.spark.sql.{DataFrame, Dataset, Encoders, SparkSession}
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.expressions.scalalang.typed.{count => typedCount}
+
+object TwitterReportGenerator {
+
+  def getReportWithSqlProcessing(spark:SparkSession,tweetStatusDf:DataFrame):DataFrame = {
+    log.debug("=======> Stage 3: Twitter Report via SQL Start.... ")
+    tweetStatusDf.createOrReplaceTempView("status")
+    val query = "select location,id,count(id) as frequency from status group by location,id having frequency >= 10"
+    spark.sql(query).withColumn("Date",lit(current_date)).createOrReplaceTempView("filteredTable")
+
+    spark.sql("select location, id, Date from (select *, row_number() " +
```

Comment:
please , use `"""..."""` for multi-line string formatting.

Reply:
Fixed..